pacu/modules/eks__enum/main.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+import argparse
+from botocore.exceptions import ClientError
+import base64
+
+module_info = {
+    'name': 'eks_enum',
+    'author': 'David Fentz',
+    'category': 'ENUM',
+    'one_liner': 'This module enumerates over EKS resources.',
+    'description': 'This module enumerates over EKS resources.',
+    'services': ['EKS'],
+    'prerequisite_modules': [],
+    'external_dependencies': [],
+    'arguments_to_autocomplete': [
+        '--regions',
+        '--verbose',
+        '--addons',
+        '--identity_provider_configs',
+        '--fargate_profiles',
+        '--all',
+    ],
+}
+
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info['description'])
+parser.add_argument('--regions', required=False, default=None, help='One or more (comma separated) AWS regions in the format "us-east-1". Defaults to all session regions.')
+parser.add_argument('--verbose', required=False, action='store_true', default=False, help='Enable verbose output')
+parser.add_argument('--addons', required=False, action='store_true', default=False, help='Enumerate EKS addons')
+parser.add_argument('--identity_provider_configs', required=False, action='store_true', default=False, help='Enumerate EKS identity provider configs')
+parser.add_argument('--fargate_profiles', required=False, action='store_true', default=False, help='Enumerate EKS fargate profiles')
+parser.add_argument('--all', required=False, action='store_true', default=False, help='Enumerate all EKS resources.')
```

Comment:
I think I would default to all resources and have the other, more specific flags override this.

Reply:
Suggestion implemented. :+1: